src/ServerConnectionAcceptor.cpp

```diff
@@ -159,26 +156,55 @@ void ServerConnectionAcceptor::processFrame(
     default: {
       transport->outputFrameOrEnqueue(
           frameSerializer->serializeOut(Frame_ERROR::unexpectedFrame()));
-      closeAndRemoveConnection(
-          connectionHandler,
-          std::move(transport),
-          std::runtime_error("invalid"));
+      error("Invalid frame received instead of SETUP/RESUME");
       break;
     }
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
```

Comment:
I use those `/ * 80` comments as logical separators, i.e. separating fields from methods, internal methods from public ones, ctors from methods, etc.  Though I do have a tendency to put in more than I should...  I can remove it if you want.